generators/chipyard/src/main/scala/ConfigFragments.scala

```diff
@@ -160,3 +154,12 @@ class WithControlCore extends Config((site, here, up) => {
 class WithChipTopCaughtReset extends Config((site, here, up) => {
   case BuildTop => (p: Parameters) => Module(new ChipTopCaughtReset()(p).suggestName("top"))
 })
+
+/**
+ * Config fragment to add a NVDLA to the SoC.
+ * Supports "small" and "large" configs only.
+ */
+class WithNVDLA(config: String) extends Config((site, here, up) => {
```

Comment:
Nit, but this feels like it belongs in the NVDLA-wrapper repo, instead of here.

Reply:
Ya, seems like a good idea to put this in a `Configs.scala` file in that repo.